syntax = "proto3";

package debate;


service Candidate {
  rpc Answer(AnswerRequest) returns (AnswerReply) {}
  rpc Elaborate(ElaborateRequest) returns (ElaborateReply) {}	
}

//1) Answer
message AnswerRequest {
	string question = 1; 
}

message AnswerReply {
	string answer = 1 ; 
}
//2) Elaborate

message ElaborateRequest {
	string topic = 1;
	repeated int32 blah_run = 2;
}

message ElaborateReply{
	string answer = 1; 
}